C#/cs01/tests/14.cs

```diff
@@ -0,0 +1,43 @@
+using NUnit.Framework;
+using Brainchild.Curriculum.Solutions;
+namespace Brainchild.Curriculum
+{
+    [TestFixture]
+    public class Test14
+    {
+        [SetUp]
+        public void SetUp()
+        {
+        }
+        [Test]
+        public void CheckingNormalHourReturningValue()
```

Comment:
why is this PR containing tests from another task ?

Reply:
I thought i should check every code block one by one, so that's why i  tested like 
that.